test/test_session.py

```diff
@@ -715,7 +715,10 @@ def test_snapshot_incompatible_with_causal_consistency(self):
             with self.client.start_session(causal_consistency=True,
                                            snapshot=True):
                 pass
-
+    def test_session_not_copyable(self):
```

Comment:
There should be a blank line before this method.

Reply:
Done.